packages/block-library/src/template-part/edit/utils/create-template-part-id.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Generates a template part Id based on slug and theme inputs.
+ *
+ * @param {string} slug the template part's slug
+ * @param {string} theme the template part's theme.
+ * @return {string|null} the template part's Id.
+ */
+export function createTemplatePartId( slug, theme ) {
```

Comment:
Hmmm, maybe we should swap the parameters? `theme` is a bigger unit than `slug`. And the order of the strings in the ID is `theme`//`slug` as well.

Reply:
fair enough 😁